site/app/views/forum/ForumThreadView.php

```diff
@@ -628,7 +628,7 @@ public function generatePostList($currentThread, $posts, $unviewed_posts, $curre
                 "merge_thread_content" => $merge_thread_content,
                 "csrf_token" => $csrf_token,
                 "activeThreadTitle" => $activeThreadTitle,
-                "post_box_id" => $post_box_id,
+                "post_box_id" => $post_box_id + 1,
```

Comment:
Is there a reason to add one to all of the objects?

Reply:
Yes. The file upload functions in drag-and-drop.js use an array that's indexed by post_box_id-1. The edit form's id had been hardcoded to 0, so this was trying to access index -1 and would throw an error. I figured the simplest fix would be to shift all the ids by 1 rather than trying to give the edit form an id based on the number of posts or rewrite the drag-and-drop code.

I can take a look and clean up how the ids are adjusted or try an alternative approach.